modules/ssh/manifests/keys.pp

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+class ssh::keys {
+    include ::config
+
+    $from_extsync = hiera('ssh-keys', {})
```

Comment:
No Hiera lookups in component modules.  All lookups should be done within the profiles.  We also don't use extsync in ronin puppet.